<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefWidgetUi</class>
 <widget class="QWidget" name="PrefWidgetUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Controller</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Status bar</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="mShowClockRateCheckBox">
          <property name="text">
           <string>Show clock rate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mShowWallTimeCheckBox">
          <property name="text">
           <string>Show wall time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mShowSimTimeCheckBox">
          <property name="text">
           <string>Show simulation time</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Wall Time Specification</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="mLocalTimeSpecButton">
           <property name="text">
            <string>Local Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="mUTC_TimeSpecButton">
           <property name="text">
            <string>Coordinated Universal Time (UTC)</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QRadioButton" name="mUTC_OffsetTimeSpecButton">
             <property name="text">
              <string>UTC Offset:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="mUTC_OffsetHrsBox">
             <property name="suffix">
              <string> hrs</string>
             </property>
             <property name="minimum">
              <number>-14</number>
             </property>
             <property name="maximum">
              <number>14</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="mUTC_OffsetMinsBox">
             <property name="suffix">
              <string> min</string>
             </property>
             <property name="maximum">
              <number>59</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>At Scenario End</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="mPause">
        <property name="text">
         <string>暂停</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mLoop">
        <property name="text">
         <string>Loop to start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mExtrapolate">
        <property name="text">
         <string>Extrapolate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Maximum Play Rate</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QComboBox" name="mSpeedLimitComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Using a high play rate in event-dense scenarios may result in a degraded experience.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
